### ISSUE TITLE
Move related class ObdDecoder and BcdUtil into package org.traccar.protocol

### ISSUE THREAD

**Issue by Joegardner11** [traccar/traccar#4976]
Hi, @tananaev 
**_Issue Description_**

Package **_org.traccar.helper_** groups together miscellaneous classes that might be useful to different packages. The package structure violates the “high cohesion and low coupling” design rules. I found that class **_ObdDecoder_** and **_BcdUtil_** strongly interact with classes in the other package (**_org.traccar.protocol_**) by checking the source code and the snippet of the dependency graph. The project has grown larger, leading to becoming increasingly hard to maintain. During the development process, one groups together classes (that often co-change) with similar responsibilities in one package to facilitate maintenance, which prevents a change that causes other packages to be modified. For example, if one modifies package **_org.traccar.helper_** (i.e., package rename), resulting in multiple classes of package **_org.traccar.protocol_** to be changed.

Location: The source file can be found at path File  src/main/java/org/traccar/helper/ObdDecoder.java
src/main/java/org/traccar/helper/BcdUtil.java

Refactoring suggestions

I suggest to move related class  **_ObdDecoder_** and **_BcdUtil_** into package **_org.traccar.protocol_**.
<img width="478" alt="BcdUtil-java" src="https://user-images.githubusercontent.com/29889989/199512231-f4695368-d218-42e0-8daa-7b188830c2a3.png">
<img width="518" alt="ObdDecoder-java" src="https://user-images.githubusercontent.com/29889989/199512263-afba9e56-8f89-4c8a-ad61-0c8c8a8af55b.png">

**Comment by tananaev** [comment]
I don't think it makes sense. Package "protocol" already has the most classes, so I want to avoid adding anything else there. Also, those are generic utilities, not specific to any protocol.

**Comment by tananaev** [comment]
Closing as the suggestion doesn't make sense in this case.
